### PR TITLE
Fix qt api issues

### PR DESCRIPTION
**Pull request by D-Triveni**
Dependent on https://github.com/nrfconnect/sdk-nrf/pull/12341

### CODE REVIEW

File: samples/wifi/wfa_qt_app/prj.conf [hunk 1]
```diff
@@ -100,3 +100,7 @@ CONFIG_SETTINGS_NVS=y
 # environment as the queue depth.
 CONFIG_NET_MGMT_EVENT_QUEUE_SIZE=150
 CONFIG_NET_SOCKETS_POSIX_NAMES=y
+
+CONFIG_SHELL_BACKEND_DUMMY=y
+CONFIG_NRF_WIFI_LOW_POWER=n
+CONFIG_WPA_SUPP_EAPOL=y
```
Comment (krish2718) [inline samples/wifi/wfa_qt_app/prj.conf:106]: Add a comment that this is needed for PMKSA caching tests

File: samples/wifi/wfa_qt_app/prj.conf [hunk 2]
```diff
@@ -100,3 +100,7 @@ CONFIG_SETTINGS_NVS=y
 # environment as the queue depth.
 CONFIG_NET_MGMT_EVENT_QUEUE_SIZE=150
 CONFIG_NET_SOCKETS_POSIX_NAMES=y
+
+CONFIG_SHELL_BACKEND_DUMMY=y
+CONFIG_NRF_WIFI_LOW_POWER=n
```
Comment (krish2718) [inline samples/wifi/wfa_qt_app/prj.conf:105]: Even this needs a comment as to why we are disabling LPM?

**Review by krish2718 — APPROVED** [review]

**Review by bama-nordic — APPROVED** [review]

File: samples/wifi/wfa_qt_app/README.rst [hunk 3]
```diff
@@ -40,6 +40,15 @@ The following is an example of the CLI command:
 
    west build -b nrf7002dk_nrf5340_cpuapp
 
+To build for the nRF7002 DK with the netusb support, use the ``nrf7002dk_nrf5340_cpuapp`` build target with netusb overlay configuration.
```
Comment (richabp) [inline samples/wifi/wfa_qt_app/README.rst:43]:
```suggestion
To build for the nRF7002 DK with the netusb support, use the ``nrf7002dk_nrf5340_cpuapp`` build target with the configuration overlay :file:`overlay-netusb.conf`.
```

**Review by richabp — CHANGES_REQUESTED** [review]
One minor comment.

**Review by richabp — APPROVED** [review]

**Pull request merged**